client/client.go

```diff
@@ -31,7 +33,140 @@ func NewCollector() *Collector {
 
 func (c *Collector) Send(cmd *cobra.Command) error {
 
-	return send(c.EventsUrl, fromCommand(cmd))
+	return send(c.EventsUrl, redact(fromCommand(cmd)))
+}
+
+func redact(telemetry *model.Telemetry) *model.Telemetry {
+
+	return redactFlags(redactArgs(telemetry))
+}
+
+func redactFlags(telemetry *model.Telemetry) *model.Telemetry {
+
+	for key := range telemetry.Flags {
+		if key == "err-ignore" ||
+			key == "warn-ignore" ||
+			key == "match-path" ||
+			key == "prefix-base" ||
+			key == "prefix-revision" ||
+			key == "strip-prefix-base" ||
+			key == "strip-prefix-revision" ||
+			key == "filter-extension" {
+			telemetry.Flags[key] = ""
+		}
+	}
+
+	return telemetry
+}
+
+func redactArgs(telemetry *model.Telemetry) *model.Telemetry {
+
+	for i := 0; i < len(telemetry.Args); i++ {
+		if telemetry.Args[i] != "" {
+			telemetry.Args[i] = getCategory(telemetry.Args[i])
+		}
+	}
+
+	return telemetry
+}
+
+func getCategory(name string) string {
+
+	if web := getWebCategory(name); web != "" {
+		return web
+	}
+	return "file"
+}
+
+func getWebCategory(name string) string {
+
+	if isSwaggerHub(name) {
+		return "swaggerhub"
+	} else if isGitHub(name) {
+		return "github"
+	} else if isGCS(name) {
+		return "gcs"
+	} else if isS3(name) {
+		return "s3"
+	} else if isAzure(name) {
+		return "azure"
+	} else if isHeroku(name) {
+		return "heroku"
+	} else if strings.HasPrefix(name, "https://") {
+		return "https"
+	} else if strings.HasPrefix(name, "http://") {
+		return "http"
+	}
+
+	return ""
+}
```

Comment:
I think `file`, `swaggerhub` and `http/https` would suffice for the source.

If other protocols like `S3` / `FTP`, ... are supported those could also be added.

Reply:
What do you mean by supported?

Reply:
I think of `file` and `http/https` as supported protocols and `swaggerhub` as special case of interest.

I don't see how knowing what hosters are used (azure/heroku/...) is useful to oasdiff and therefore merge all of them into `http/https`. If you think that this information is useful keeping it would be ok as well - the important part was removing the full url.

With supported protocols i mean cases that are not a plain http requests, to give an example for s3:
Imagine `oasdiff` accepting a argument like `s3://<bucket-name>.s3.amazonaws.com/specs` to compare all files in a local directory with all files in a bucket in `composed mode`.